[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\sbacelar\\AppData\\Local\\Temp\\quarto-sessionf87fcad6937504c4'\n"]
